cmd/ecloud/ecloud_instance.go

```diff
@@ -212,6 +213,7 @@ func ecloudInstanceUpdateCmd(f factory.ClientFactory) *cobra.Command {
 	cmd.Flags().String("name", "", "Name of instance")
 	cmd.Flags().Int("vcpu", 0, "Number of vCPU cores to allocate")
 	cmd.Flags().Int("ram", 0, "Amount of RAM (in MB) to allocate")
+	cmd.Flags().String("volumegroup", "", "ID of volumegroup to use for instance")
```

Comment:
Personal preference - `volume-group` just for consistency with other flags elsewhere

Reply:
I was kinda debating what looked better.  I'll get these updated for consistency :)